docs/assets/javascripts/version-select.js

```diff
@@ -37,9 +39,51 @@ window.addEventListener("DOMContentLoaded", function() {
         var deprecatedVersionsArray = new Array();
         var versionsArray = new Array();
 
+        // Used to decorate options with a numeric sort order
+        function decorateWithSortOrder(option) {
+            // Option version, formatted as major.minor.patch (e.g. 1.1.0)
+            var optionVersion = option.value.match(/[\d.]+/);
+            optionVersion = optionVersion ? optionVersion[0] : '';
+
+            // We split over an array for further processing.
+            var optionVersionArray = optionVersion.split('.').map(function (v) {
+                return parseInt(v, 10);
```

Comment:
What does this do? 🤔

Reply:
The same as `parseInt`. The second argument is the radix (in case you wanted to parse a hex string). I think the default is 10, so I could have omitted it.

But basically this code transforms `10.1.456` into `['10', '1', '456']` into `[10, 1, 456]`